core/src/main/scala/scalaz/Order.scala

```diff
@@ -85,20 +85,32 @@ object Order {
 
   ////
 
-  implicit val orderInstance: Divisible[Order] = new Divisible[Order] {
-    override def contramap[A, B](r: Order[A])(f: B => A) = r.contramap(f)
-
-    override def conquer[A] = order((_, _) => Ordering.EQ)
-
-    override def divide2[A, B, C](fa: =>Order[A], fb: =>Order[B])(f: C => (A, B)) =
-      order[C]{ (c1, c2) =>
-        val (a1, b1) = f(c1)
-        val (a2, b2) = f(c2)
-        fa.order(a1, a2) match {
-          case Ordering.EQ => fb.order(b1, b2)
-          case o => o
+  implicit val orderDecidableInstance: Decidable[Order] = new OrderDecidableInstance
+
+  private[scalaz] class OrderDecidableInstance extends Decidable[Order] with Divisible[Order] {
```

Comment:
it's ok, let's not bother with creating classes anymore... I don't think it is going to work out.